blog/_posts/2023-01-24-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,245 @@
+
+## Roadmap for 2023 (and Beyond)
+
+It is now time to look forward and present our goals for 2023. Our mission remains the same: improve the experience of
+becoming and being a Scala developer, and help the community to build a rich ecosystem of libraries. Ultimately, our
+work will ensure the longevity of Scala by creating a reliable and resilient ecosystem.
+
+We have identified the priorities for 2023 through our discussions with the community (online or at conferences),
+with our [Advisory Board members](https://scala.epfl.ch/#advisory-board-member-list), and with the
+main organizations that are [behind Scala](https://www.scala-lang.org/community/#whos-behind-scala)
+([LAMP](https://lamp.epfl.ch), [Lightbend](https://lightbend.com), and [VirtusLab](https://virtuslab.com)). We are
+grateful to all of them.
+
+In the following subsections we remind our ongoing and recurring projects, and we present our most important goals as
+well as some additional stretch goals that would need more resources.
+
+The roadmap we present here is of course subject to adjustments throughout the year.
+
+### Language, Compiler, Standard Library
+
+In addition to keeping the Scala Improvement Process going, and reducing the number of bugs in the compiler and standard
+library, we will:
+
+- **Make the compiler error messages clearer and more actionable.** The compiler should help developers write correct code
+  instead of “just complaining” about incorrect code. You can find a list of related issues
+  [here](https://github.com/lampepfl/dotty/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%3Areporting).
+- **Bring the compiler’s suggestions into the IDEs.** Instead of emitting only text output, the compiler
+  will produce structured output that IDEs (Metals and IntelliJ) will be able to read to automatically provide “quick
+  fixes” to the users. This should reduce the implementation (and maintenance) costs on the IDE side.
+- **Unfreeze the Scala standard library.** The standard library has not changed (except for bug fixes or some performance
+  improvements) since Scala 2.13, which was released almost four years ago. [Dropping the forwards binary compatibility
+  requirement](https://github.com/scala/improvement-proposals/pull/54) would be a first step to allow the introduction
+  of new classes or the addition of new members to the existing classes. Then, we will create a process to validate
+  what should be in the standard library and what should stay outside of it.
+- **Clarify and communicate the Scala 3 roadmap.** We will coordinate with the compiler team to clarify what will
+  go into the [Long-Term Support version]({% link blog/_posts/2022-08-17-long-term-compatibility-plans.md %})
+  of Scala 3, and make sure that roadmap is well communicated. We will focus primarily on the items that will
+  enable more users to migrate from Scala 2 to Scala 3 (e.g., missing compiler options).
```

Comment:
I think that linting options should be additionally mentioned directly in the "Make the compiler error messages clearer and more actionable.", because the term "missing compiler options" is too vague.

Reply:
OK, done in 4865541e